eng/stylecop.json

```diff
@@ -16,4 +16,3 @@
         }        
     }
   }
-  
```

Comment:
revert

Reply:
I can not figure out why vs 2019 keeps addinig that blank line

Reply:
don't open VS when you revert :)